lte/gateway/c/core/oai/test/amf/BUILD.bazel

```diff
@@ -37,6 +37,39 @@ cc_test(
     ],
 )
 
+cc_test(
+    name = "amf_procedures_test",
+    size = "small",
+    srcs = [
+        "test_amf_procedures.cpp",
+    ],
+    # TODO: Remove manual tag when fixed: GH12144
+    tags = ["manual"],
+    deps = [
+        ":amf_app_test_util",
+        "//lte/gateway/c/core",
+        "//lte/gateway/c/core/oai/test/mock_tasks",
+        "@com_google_googletest//:gtest_main",
+    ],
+)
+
+cc_test(
+    name = "amf_stateless_test",
+    size = "small",
+    srcs = [
+        "test_amf_common_utils.cpp",
```

Comment:
Are the two test files dependent? Or should `test_amf_common_utils.cpp` be modeled by a separate test?

Reply:
Nice find! I pulled out the `test_amf_common_utils.cpp` into it's own test. Thank you!